### PR TITLE
Implement kernel symbol translation for BPFd

### LINKED ISSUE #6: tools/cachetop doesn't show hits/misses counter on arm64

### PR DESCRIPTION
**Pull request by jcanseco**
Fixes #6

### CODE REVIEW

**Comment by jcanseco** [comment]
This is dependent on PR #23  and #24.

**Comment by jcanseco** [comment]
This is dependent on PR #23 and #24.

File: INSTALL.md [hunk 1]
```diff
@@ -101,19 +101,53 @@ make
...
+./configure --host=aarch64-linux-gnu --prefix=/usr/aarch64-linux-gnu/ --enable-maintainer-mode
+mv ~/i386_gendis ./libcpu/i386_gendis
+make
+sudo make install
+```
+
+Once all the dependencies are ready, you can then finally build BPFd for arm64:
 ```
 git clone git@github.com:joelagnel/bpfd.git; cd bpfd
 mkdir -p build; cd build
```
Comment (joelagnel) [inline INSTALL.md]: Don't you need to update this part so that it uses cmake to generate the Makefiles?

File: INSTALL.md [hunk 2]
```diff
@@ -101,19 +101,53 @@ make
...
+autoreconf -i -f
+
+# Build elfutils for x86 first since we need an x86 copy of i386_gendis to build elfutils for arm64
+./configure --enable-maintainer-mode
+make
+cp ./libcpu/i386_gendis ~/
+
+# Build elfutils for arm64
+./configure --host=aarch64-linux-gnu --prefix=/usr/aarch64-linux-gnu/ --enable-maintainer-mode
+mv ~/i386_gendis ./libcpu/i386_gendis
```
Comment (joelagnel) [inline INSTALL.md]: Jazel, does the Usym lookup work correctly with the elfutils you built?

Comment (joelagnel) [reply INSTALL.md]: Oh never mind, I remembered you mentioned you need to build it for ksym lookup to work as well.

File: INSTALL.md (hunk 1 above)
Comment (joelagnel) [reply INSTALL.md]: Ignore my comment. I saw that's a part of a different PR.

**Comment by jcanseco** [comment]
I went ahead and rebased this branch on top of 'cmake'.

**Review by joelagnel — APPROVED** [review]

**Pull request merged**